packages/typespec-vscode/src/emit/emit-quick-pick-item.ts

```diff
@@ -0,0 +1,11 @@
+import vscode from "vscode";
+import { EmitterKind } from "./emitter.js";
+
+export interface EmitQuickPickItem extends vscode.QuickPickItem {
```

Comment:
feels you dont need to define this type explicitly either.

Reply:
The `EmitQuickPickItem` has several extra properties, it is not simple one, so I think we can define an explicit type here to clear understanding and good readability in future maintenance.